src/background/lib/api.js

```diff
@@ -17,11 +17,11 @@ export const generateDataUri = async (url, callback) => {
     canvas.getContext('2d').drawImage(this, 0, 0)
     callback(url, canvas.toDataURL('image/png'))
   }
-  let formattedUrl = (url.substring(0,2) == '//' ? "https:":'' ) + url
-  if(formattedUrl.substring(0,1) == '/') {
+  let formattedUrl = (url.substring(0, 2) === '//' ? 'https:' : '') + url
+  if (formattedUrl.substring(0, 1) === '/') {
     chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
```

Comment:
@dannysummerlin the tag object is given when the user initiates clipping in `chrome.contextMenus.onClicked` and `chrome.browserAction.onClicked`. To make this a little cleaner, can you use that passed variable?